nuttx-configs/px4fmu-v4pro/nsh/defconfig

```diff
@@ -627,7 +627,7 @@ CONFIG_BOOT_RUNFROMFLASH=y
 # Boot Memory Configuration
 #
 CONFIG_RAM_START=0x20000000
-CONFIG_RAM_SIZE=327680
```

Comment:
@davids5 
Actually the parameter is used in load monitor : https://github.com/PX4/Firmware/blob/master/src/modules/load_mon/load_mon.cpp#L243-L245

If the parameter is not right, it triggers an overload alert in commander : https://github.com/PX4/Firmware/blob/master/src/modules/commander/commander.cpp#L3137

Do you agree?